src/api/telephony/spare/telephony-spare.v6.service.js

```diff
@@ -49,6 +49,11 @@ angular.module('ovh-api-services').service('OvhApiTelephonySpareV6', ($resource,
       method: 'POST',
       url: '/order/telephony/spare/new',
     },
+    getCompatibleReplacement: {
```

Comment:
please use `query` instead of `get` if `isArray: true`:

`queryCompatibleReplacement`

:)